### PR TITLE
Refactor usage worker sn

### PR DESCRIPTION
**Pull request by shimonewman**
## What does this PR do?

(Provide a description of what this PR does and why it's needed.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: src/Appwrite/Platform/Workers/Usage.php [hunk 1]
```diff
@@ -65,16 +64,14 @@ public function action(Message $message, $pools, $cache): void
...
                 document: new Document($document),
                 metrics:  $payload['metrics'],
-                pools: $pools,
-                cache: $cache
+                getProjectDB: $getProjectDB
             );
         }
 
-        self::$stats[$projectId]['database'] = $project->getAttribute('database');
+        self::$stats[$projectId]['project'] = $project;
```
Comment (Meldiron) [inline src/Appwrite/Platform/Workers/Usage.php:74]: I dont know what this change does. If fine by you, then fine by me.

File: src/Appwrite/Platform/Workers/UsageHook.php [hunk 2]
```diff
@@ -23,83 +24,71 @@ public function __construct()
...
-                        $cache
-                    );
-
-                    $dbForProject->setNamespace('_' . $projectInternalId);
-
-                    foreach ($project['keys'] ?? [] as $key => $value) {
-                        if ($value == 0) {
-                            continue;
-                        }
+            $interval = (int) App::getEnv('_APP_USAGE_AGGREGATION_INTERVAL', '50000');
```
Comment (Meldiron) [inline src/Appwrite/Platform/Workers/UsageHook.php:42]: This was 30s before, and is now 5s. Just making sure you are aware.

File: src/Appwrite/Platform/Workers/UsageHook.php [hunk 3]
```diff
@@ -23,83 +24,71 @@ public function __construct()
...
+            Timer::tick($interval, function () use ($register, $getProjectDB) {
+                $offset = count(self::$stats);
+                $projects = array_slice(self::$stats, 0, $offset, true);
+                array_splice(self::$stats, 0, $offset);
+                var_dump($projects);
```
Comment (Meldiron) [inline src/Appwrite/Platform/Workers/UsageHook.php:47]: Leftover

File: src/Appwrite/Platform/Workers/UsageHook.php (hunk 2 above)
Comment (shimonewman) [reply src/Appwrite/Platform/Workers/UsageHook.php:42]: its 50 sec no?

File: src/Appwrite/Platform/Workers/UsageHook.php (hunk 3 above)
Comment (shimonewman) [reply src/Appwrite/Platform/Workers/UsageHook.php]: done

File: .env [hunk 4]
```diff
@@ -75,7 +75,7 @@ _APP_MAINTENANCE_RETENTION_CACHE=2592000
 _APP_MAINTENANCE_RETENTION_EXECUTION=1209600
 _APP_MAINTENANCE_RETENTION_ABUSE=86400
 _APP_MAINTENANCE_RETENTION_AUDIT=1209600
-_APP_USAGE_AGGREGATION_INTERVAL=5
+_APP_USAGE_AGGREGATION_INTERVAL=50000
```
Comment (Meldiron) [inline .env:78]: Regarding above, then this change goes from 5s to 50s.

Comment (shimonewman) [reply .env:78]: yeah, I used an env that was used to aggregate via influx
Updated it to 60 sec (Eldad suggested)
Once it works we can tweak the sec

**Review by Meldiron — APPROVED** [review]

**Pull request merged**